React/Views/ScrollView/RCTScrollView.m

```diff
@@ -1175,7 +1176,10 @@ - (void)updateContentSizeIfNeeded
 {
   CGSize contentSize = self.contentSize;
   if (!CGSizeEqualToSize(_scrollView.contentSize, contentSize)) {
-    _scrollView.contentSize = contentSize;
+    [_scrollView setContentSize:contentSize];
```

Comment:
Technically an iOS change, can we mark it with a diff tag or ifdef if need be?

Reply:
Ah, I'll revet this. Was trying to make it obvious that calling the setter has side effects (scrolling to the correct offset)

Reply:
reverted